include/alpaka/onHost/Device.hpp

```diff
@@ -26,6 +26,7 @@ namespace alpaka::onHost
         friend struct internal::GetNativeHandle;
 
         using element_type = typename DeviceHandle::element_type;
+        using DeviceKind = T_DeviceKind;
```

Comment:
this should not be required there is a  `getDeviceKind(any)` you could  use. The type can be queried via
`ALPAKA_TYPEOF(getDeviceKind(any))`